diffkemp/diffkabi.py

```diff
@@ -195,9 +195,14 @@ def text_indent(text, width):
                     called_res.second.callstack.replace(dest_path + "/", ""),
                     indent))
                 output.write("\n\n")
-            output.write(text_indent("Diff:\n", indent))
-            output.write(text_indent(
-                called_res.diff, indent))
+            if called_res.diff.strip() == "" and \
+               called_res.macro_diff is not None:
```

Comment:
Use parentheses around multiline if condition (this is specified by PEP I think).

Reply:
Interestingly flake8 does not detect this, despite the fact this surely is specified by PEP.